collect/collect_test.go

```diff
@@ -194,7 +194,7 @@ func TestOriginalSampleRateIsNotedInMetaField(t *testing.T) {
 		GetTracesConfigVal: config.TracesConfig{
 			SendTicker:   config.Duration(2 * time.Millisecond),
 			SendDelay:    config.Duration(1 * time.Millisecond),
-			TraceTimeout: config.Duration(60 * time.Second),
+			TraceTimeout: config.Duration(1 * time.Second),
```

Comment:
Why do we change this to 1 second?

Reply:
The non-sampled event is added and we only waits 5 seconds for it to appear in the transmission queue. The cache will only expire and send the events once the trace timeout is reached.

I'm not sure how / why it currently works in main.

Reply:
Oh, it's because the cache size is set to `3` in the `cache.NewInMemCache()`. Once the cache is full, traces will be ejected from the cache when a new span comes in and put into the transmission queue.